### PR TITLE
add hello world example

### PR DESCRIPTION
**Pull request by manyuanrong**
#139

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+//! let platform = platform::new_default_platform();
+//! V8::initialize_platform(platform);
+//! V8::initialize();
+//!
+//! let mut create_params = Isolate::create_params();
+//! create_params.set_array_buffer_allocator(Allocator::new_default_allocator());
+//! let isolate = Isolate::new(create_params);
+//! let mut locker = Locker::new(&isolate);
+//!
+//! HandleScope::enter(&mut locker, |scope| {
```
Comment (bartlomieju) [inline src/lib.rs]:
```
let mut hs = v8::HandleScope::new(&mut locker);
let scope = hs.enter();
```

Comment (manyuanrong) [reply src/lib.rs]: > ```
> let mut hs = v8::HandleScope::new(&mut locker);
> let scope = hs.enter();
> ```

 But I can't find `new` in`HandleScope`

**Comment by manyuanrong** [comment]
I created a project of Hello world to ensure that the sample code can run normally
https://github.com/manyuanrong/rusty_v8_hello_world

File: src/lib.rs (hunk 1 above)
Comment (bartlomieju) [reply src/lib.rs]: https://github.com/denoland/rusty_v8/blob/29fa5388f547e16596f71bfb2f0fd8e6b7021c2d/src/handle_scope.rs#L35-L54

Comment (manyuanrong) [reply src/lib.rs]: OK, I saw it in this project, but my example uses the version released on "create.io", which seems to be older.

File: src/lib.rs [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+//! code.to_rust_string_lossy(scope);
+//! let mut script = Script::compile(scope, context, code, None).unwrap();
+//! let result = script.run(scope, context).unwrap();
+//! let result: Local<rusty_v8::String> = unsafe { std::mem::transmute_copy(&result) };
+//!
+//! let str = result.to_rust_string_lossy(scope);
+//!
+//! println!("{}", str);
+//!
+//! context.exit();
```
Comment (bartlomieju) [inline src/lib.rs]: Can you wrap this bit in curly braces (to showcase how long `HandleScope` lives). Also please add `drop(locker)` at the end.

**Comment by ry** [comment]
@manyuanrong thanks!

**Pull request merged**